extension/experiments/voice/api.js

```diff
@@ -54,6 +54,15 @@ this.voice = class extends ExtensionAPI {
           async quitApplication() {
             return runCommand("cmd_quitApplication");
           },
+
+          async openPreferences() {
+            return runCommand("Browser:OpenAboutContainers");
+          },
+
+          async browserOpenAddonsMgr() {
+            return runCommand("Tools:Addons");
```

Comment:
I kind of wish we had a general way to open `about:` URLs. I've done that before, but I'm a little afraid that the code is fragile, or might even be broken right now: https://github.com/ianb/sticky-reader-mode/blob/master/addon/experiment/api.js

What you've done is probably safer for now.

Reply:
OK cool. It will come in very handy when fixing. about:debugging